.travis/obs.sh

```diff
@@ -1,13 +1,11 @@
 #!/bin/bash
 
-[ "${TRAVIS_PULL_REQUEST}" == "false" -a \
-  "${TRAVIS_BRANCH}" == "${MY_DEPLOY_BRANCH}" \
-] || exit
```

Comment:
You should reintroduce this so that it only runs for the master branch.

Or, probably better, use `if:` expressions in ci.yml https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsif